gensrc/proto/olap_file.proto

```diff
@@ -129,6 +129,7 @@ message RowsetMetaPB {
     // If enable_segments_file_size is false,
     // the segments_file_size maybe is empty or error
     optional bool enable_segments_file_size = 1004;
+    optional bool has_variant_type_in_schema = 1005;
```

Comment:
Is it necessary to add this field, since it can be deduced from tablet schema?

Reply:
deduce from protobuf is trivial

Reply:
RowsetMeta lost  tablet schema info but we still need to know if variant exist in later read schema